lib/Models/CatalogItem.js

```diff
@@ -56,9 +58,9 @@ var CatalogItem = function(terria) {
     /**
      * Gets or sets the URL of the legend for this data item, or undefined if this data item does not have a legend.
      * This property is observable.
-     * @type {String}
+     * @type {Array<LegendUrl>}
      */
-    this.legendUrl = undefined;
+    this._legendUrls = undefined;
```

Comment:
Now that this is a private property, the doc shouldn't go here, because the doc ends up looking like this:

![image](https://cloud.githubusercontent.com/assets/924374/12134273/f24e2d7e-b481-11e5-93db-dd7fb9c1a450.png)

Document the public property instead.